spec/controllers/wishlists_controller_spec.rb

```diff
@@ -100,20 +100,72 @@
         it "redirects to the wishlists index page" do
           is_expected.to redirect_to(wishlists_path)
         end
+      end
+    end
+
+    describe "GET #edit" do
+      context "when wishlist exists" do
+        subject! { get_edit(id: wishlist.id) }
```

Comment:
it should be `before` block instead. You are not using `subject` in assertion below.